src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -29,20 +29,27 @@
 * @date 2011 (1.x)
 * @group Accounts
 * @group-content ../../ApexDocContent/Accounts.htm
-* @description Implements Household or Administrative account model - controls syncing within the model
+* @description Implements Household or Administrative account model - controls syncing within the model i.e.,
+* 1) It ensures that when the owner of a Primary Contact record that has an Administrative or
+* Household Account is updated, the owner of the Account is also updated. 
+* 2) It also ensures that when a Primary Contact is moved from an Administrative or Household Account, 
+* the first remaining Contact on the Account becomes the Account's Primary Contact.
+* 3) When the name of a Contact used to name an Account changes, the Account's name is updated using the 
+* Account Naming Format specified on the Systems tab in EDA Settings. These behaviors apply to all Accounts
+* with the Administrative or Household Account Record Type specified on the "Accounts and Contacts" tab in "EDA Settings.”
 */
 public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
     /*******************************************************************************************************
     * @description The selected Account record type.
     */
     private static ID defaultRecTypeId = UTIL_CustomSettingsFacade.getSettings().Account_Processor__c;
 
-    /*******************************************************************************************************
-    * @description The user defined Administrative Account record type.
+    /*****************************************************************************************************************
+    * @description The user defined Administrative Account record type in "ACCOUNTS AND CONTACTS" tab of EDA Settings.
     */
-    private static ID userDefinedAdminRecordTypeId = Util_Describe.getAdminAccRecTypeID();
+    private static ID userDefinedAdminRecordTypeId = Util_Describe.getCustomAdminAccRecTypeId();
```

Comment:
This is the only logic change in this class, other changes are to better name variables and spacing fixes.